python/ray/experimental/state/common.py

```diff
@@ -493,17 +559,25 @@ class WorkerState(StateSchema):
     #: -> worker_launched_time_ms (process started).
     #: -> start_time_ms (worker is ready to be used).
     #: -> end_time_ms (worker is destroyed).
-    worker_launch_time_ms: Optional[int] = state_column(filterable=False, detail=True)
+    worker_launch_time_ms: Optional[int] = state_column(
+        filterable=False, detail=True, metadata={"format_fn": Humanify.timestamp}
+    )
     #: The time worker is succesfully launched
     #: -1 if the value doesn't exist.
-    worker_launched_time_ms: Optional[int] = state_column(filterable=False, detail=True)
+    worker_launched_time_ms: Optional[int] = state_column(
+        filterable=False, detail=True, metadata={"format_fn": Humanify.timestamp}
```

Comment:
nit: I thought we mentioned we could just do?
```suggestion
        filterable=False, detail=True, format_fn: Humanify.timestamp}
```

We could add `format_fn` in the `m` here directly: https://github.com/ray-project/ray/pull/35059/files#diff-9a5585a958c38ab1e7b4e84e974a5d30352a2943615f48a0322de87fd675dd1aL158

Reply:
Right, that makes the code cleaner. I refactored state_column to include format_fn.